ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/rlpx/RlpxAgent.java

```diff
@@ -67,7 +68,9 @@
   private final PeerRlpxPermissions peerPermissions;
   private final PeerPrivileges peerPrivileges;
   private final int maxConnections;
+  private final boolean randomPeerPriority;
   private final int maxRemotelyInitiatedConnections;
+  private final Bytes nodeIdMask = Bytes.random(SECP256K1.PublicKey.BYTE_LENGTH);
```

Comment:
I'd add some commentary about the purpose.  Something about the scope (per agent) and purpose (to thwart external gaming of the random comparison).